near-contract-standards/src/event.ts

```diff
@@ -0,0 +1,19 @@
+import { near } from "near-sdk-js";
+
+export class NearEvent {
+  internal_to_json_string(): string {
```

Comment:
Should it be private? Looks like it is not intended to be executed by the user.

Reply:
You're right. Making it private